carml/1.3.0/Microsoft.Storage/storageAccounts/deploy.bicep

```
@@ -73,7 +73,7 @@ param networkAcls object = {}
 @description('Optional. A Boolean indicating whether or not the service applies a secondary layer of encryption with platform managed keys for data at rest. For security reasons, it is recommended to set it to true.')
 param requireInfrastructureEncryption bool = true
 
-@description('Optional. Allow or disallow cross AAD tenant object replication.')
+@description('Optional. Allow or disallow cross EntraID tenant object replication.')
```

When used with "tenant" it's just "Microsoft Entra" not "Microsoft Entra ID" (phew, I know these "rules" are complex)

```suggestion
@description('Optional. Allow or disallow cross Microsoft Entra tenant object replication.')
```